docs/commerce/payment/payment.md

```diff
@@ -15,8 +15,10 @@ From the development perspective, the component enables customization of the pay
 The component exposes the following:
 
 - [Payment method PHP API](payment_method_api.md) that allows for managing payment methods
+- [Payment method REST API](../../api/rest_api/rest_api_reference/rest_api_reference.html#orders) that helps manage payment methods over HTTP
```

Comment:
It should link to https://doc.ibexa.co/en/latest/api/rest_api/rest_api_reference/rest_api_reference.html#payment-methods